### DISCUSSION TITLE
Ability to gift Dinos

### DISCUSSION THREAD

**Discussion by zerafox** [gameserverapp/Platform#662] [Suggestions]
It would be nice to be able to give tamed dinos to users, just being able to set the gender (or random), a level range (or specific level) and whether or not that creature is neutered would be lovely!

I would like to be able to create a "Starter Pack Dino" for new players that they could get a tame early on but I wouldn't want it to be able to breed, I'd like to choose the level, possible gender and so forth.

I realize that this would also have to be mod-side as well, as ideally this would give the player an item that they could use to spawn said creature into the world.

**Comment by mvaessen** [comment]
Hi, there are mods available that will let you do this I guess. But this is not something I'm planning to add to the gsa mod or system. Too much maintenance required. I recommend checking out the mods in the public-updates on our discord
